src/codegen_shared.h

```diff
@@ -147,4 +147,4 @@ static inline llvm::Value *get_current_ptls_from_task(llvm::IRBuilder<> &builder
     auto ptls = CastInst::Create(Instruction::BitCast, ptls_load, T_ppjlvalue, "ptls");
     builder.Insert(ptls);
     return ptls;
-}
```

Comment:
Why this removed the newline (to an otherwise untouched file)?

Reply:
Looks like it was left over from when I investigated adding write barriers to these loops, I'll get rid of it.